src/main/java/com/mitchej123/hodgepodge/mixins/ic2/MixinTileEntityReactorChamberElectricInvSpeedup.java

```diff
@@ -0,0 +1,41 @@
+package com.mitchej123.hodgepodge.mixins.ic2;
+
+import ic2.api.Direction;
+import ic2.core.block.reactor.tileentity.TileEntityNuclearReactorElectric;
+import ic2.core.block.reactor.tileentity.TileEntityReactorChamberElectric;
+import net.minecraft.tileentity.TileEntity;
+import org.spongepowered.asm.mixin.Mixin;
+import org.spongepowered.asm.mixin.injection.At;
+import org.spongepowered.asm.mixin.injection.Inject;
+import org.spongepowered.asm.mixin.injection.Redirect;
+import org.spongepowered.asm.mixin.injection.callback.CallbackInfoReturnable;
+
+@Mixin(TileEntityReactorChamberElectric.class)
+public class MixinTileEntityReactorChamberElectricInvSpeedup {
+    private TileEntityNuclearReactorElectric master;
```

Comment:
This should be a weak reference in case the reactor gets updated/broken/etc.

Reply:
If reactor got updated the chambers would pop off as items

Reply:
In any case it probably won't hurt to use a weak reference instead.